src/simulation/test/LoadGeneratorTests.cpp

```diff
@@ -846,6 +830,16 @@ TEST_CASE("Multi-op mixed transactions are valid", "[loadgen]")
             cfg.LOADGEN_OP_COUNT_FOR_TESTING = {3};
             cfg.LOADGEN_OP_COUNT_DISTRIBUTION_FOR_TESTING = {1};
             cfg.TESTING_UPGRADE_MAX_TX_SET_SIZE = 1000;
+            // Calculate number of accounts needed
+            uint32_t txRate = 5;
+            uint32_t numAccounts =
+                txRate *
+                static_cast<uint32>(
+                    std::chrono::duration_cast<std::chrono::seconds>(
+                        std::chrono::seconds(5)) // Expected ledger close time
```

Comment:
I think expected close time is 1 second, iirc `Simulation::OVER_LOOPBACK` speeds up ledger close time.